src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  startingTodos: Todo[];
+  activeTodos: number;
+  selectedFilter: string;
+  setSelectedFilter: React.Dispatch<React.SetStateAction<string>>;
+  handleTodoFiltering: (v: string) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  selectedFilter,
+  activeTodos,
+  handleTodoFiltering,
+  setSelectedFilter,
+  startingTodos,
+}) => {
+  return (
+    <footer
+      className={classNames('todoapp__footer', {
+        'is-hidden': startingTodos.length === 0,
+      })}
+      data-cy="Footer"
+    >
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodos} items left
+      </span>
+
+      {/* Active link should have the 'selected' class FILTERMB*/}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: selectedFilter === '#/',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => {
+            setSelectedFilter('#/');
+            handleTodoFiltering('all');
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method